interviews.html

```diff
@@ -13,6 +13,12 @@ <h1>
 <p>(Back to my <a href="index.html">personal home page</a>.)
 
   <ul>
+    <li>
+      <a href="https://www.youtube.com/watch?v=-DVyjdw4t9I">
+        Guido van Rossum: Python and the Future of Programming | Lex Fridman Podcast #341</a>
+      (by Lex Fridman, Nov 26, 2022; <b>video podcast</b>)
+    </li>
+    
```

Comment:
Could you also add a link to the earlier one, `https://www.youtube.com/watch?v=ghwaIiE3Nd8`?

Reply:
Of course, it's done now.